cmd/cluster.go

```diff
@@ -20,3 +22,20 @@ are hopefully coming soon.'`,
 func init() {
 	rootCmd.AddCommand(clusterCmd)
 }
+
+// validateClusterInArgumentExists checks if a cluster name is present in arguments and the cluster can be found
+func validateClusterInArgumentExists(cmd *cobra.Command, args []string) error {
+
+	name := args[0]
+
+	if name == "" {
```

Comment:
This should be omitted if we force cobra command to 1 positional argument using `Args` configuration, like:
```go
var clusterCmd = &cobra.Command{
	Use:   "cluster",
	// ...
	Args: cobra.ExactArgs(1),
	// ...
}
```
We should also use a more sophisticated validation adding `ValidArgs`, that report the list of available cluster we have, eg:
```go
var clusterCmd = &cobra.Command{
	Use:   "cluster",
	// ...
	ValidArgs: AppConf.Config.AllClustersName(),
	Args: cobra.ExactValidArgs(1),
	// ...
}
```

Reply:
That is a good hint. I will try to solve it partially when I can, but actually, this is a little bit out of the scope of this PR. I think replacing all validations (because this file only defines the validation function, but the `cluster` root command doesn't use it. Other commands already do. But yes, we should do this at some point.